docs/user-guide/satisfy.md

```diff
@@ -1,3 +1,12 @@
+```{role} cvl(code)
```

Comment:
isn't this added automatically according to the conf?

Reply:
Only to the `.rst` files. I haven't yet managed to find how to add it automatically to `.md` files.